crates/runix/src/arguments/mod.rs

```diff
@@ -155,6 +155,31 @@ pub struct StoreGcArgs {
     pub max: Option<Max>,
 }
 
+/// `nix copy` options
+#[derive(Debug, Default, Clone, ToArgs)]
+pub struct CopyArgs {
+    pub from: Option<CopyFrom>,
+    pub to: Option<CopyTo>,
+}
+
```

Comment:
Optional nit: the other commands put the arg struct after the structs for each flag